expense-rescript/src/components/IncomeExpenseForm/IncomeExpenseForm.res

```diff
@@ -0,0 +1,26 @@
+%%raw("import './IncomeExpenseForm.css'")
+@react.component
+let make = () => {
+  let {transactions} = React.useContext(GlobalContext.context)
+  let income = Belt.Array.reduce(transactions, 0, (acc, value: GlobalContext.transaction) =>
+    value.price >= 0 ? acc + value.price : acc + 0
+  )
+  let finalIncome = Belt.Int.toString(income)
+
+  let expense = Belt.Array.reduce(transactions, 0, (acc, value: GlobalContext.transaction) =>
+    value.price < 0 ? acc + value.price : acc + 0
+  )
+  let finalExpense = Belt.Int.toString(expense)
```

Comment:
No need to create another variable for this conversion. Just do this wherever required